ts_scripts/spellcheck_conf/spellcheck.yaml

```diff
@@ -15,3 +15,4 @@ matrix:
   sources:
   - 'docs/*.md'
   - 'examples/*/*.md'
+  - 'examples/*/*/*.md'
```

Comment:
I believe examples/**.md would work better (did not test)

Reply:
Tried it. It doesn't work